tests/components/devolo_home_network/test_config_flow.py

```diff
@@ -125,6 +123,7 @@ async def test_zeroconf(hass: HomeAssistant) -> None:
         CONF_IP_ADDRESS: IP,
         CONF_PASSWORD: "",
     }
+    assert result2["result"].unique_id == info["serial_number"]
```

Comment:
I would prefer to have the assertion more explicit
```suggestion
    assert result2["result"].unique_id == "1234567890"
```

Reply:
It would also be nice to assert the `result["type"]`